lang/nl.json

```diff
@@ -0,0 +1,1010 @@
+        "pdp-retail-price-label": "Adviesprijs:",
+        "pdp-custom-fields-tab-label": "Aanvullende informatie"
+    },
+    "consent_manager": {
+        "data_collection_warning": "Wij gebruiken cookies (en andere soortgelijke technologieën) om gegevens te verzamelen en zo uw winkelervaring te verbeteren.",
+        "accept_all_cookies": "Alle cookies aanvaarden",
+        "gdpr_settings": "Instellingen",
+        "data_collection_preferences": "Voorkeuren voor het verzamelen van gegevens op de website",
+        "use_data_by_cookies": "gebruikt door cookies en JavaScript-bibliotheken verzamelde gegevens om uw winkelervaring te verbeteren.",
+        "data_categories_table": "In de onderstaande tabel wordt per categorie aangegeven hoe wij deze gegevens gebruiken. Om een categorie van gegevensverzameling uit te schakelen, selecteert u 'Nee' en slaat u uw voorkeuren op.",
+        "allow": "Toestaan",
+        "accept": "accepteren",
+        "deny": "Weigeren",
+        "dismiss": "Sluiten",
+        "reject_all": "Alles afwijzen",
+        "category": "Categorie",
+        "purpose": "Doel",
+        "functional_category": "Functioneel",
+        "functional_purpose": "Maken extra functies mogelijk, zoals video's en livechat. Als u deze niet toestaat, werken sommige of al deze functies mogelijk niet correct.",
+        "analytics_category": "Analyses",
+        "analytics_purpose": "Leveren statistieken over websitegebruik, bijvoorbeeld webanalytics, zodat we deze website steeds kunnen verbeteren.",
+        "targeting_category": "Targeting",
+        "advertising_category": "Reclame",
+        "advertising_purpose": "Worden gebruikt om profielen aan te maken of inhoud te personaliseren om uw winkelervaring te verbeteren.",
+        "essential_category": "Essentieel",
+        "esential_purpose": "Essentieel voor het functioneren van de site en de gevraagde diensten, maar voeren geen extra of secundaire functies uit.",
+        "yes": "Ja",
+        "no": "Nee",
+        "not_available": "N.v.t.",
+        "cancel": "Annuleren",
+        "save": "Opslaan",
+        "back_to_preferences": "Terug naar Voorkeuren",
+        "close_without_changes": "U heeft niet-opgeslagen wijzigingen aan uw voorkeuren voor gegevensverzameling. Weet u zeker dat u wilt afsluiten zonder op te slaan?",
+        "unsaved_changes": "U heeft niet-opgeslagen wijzigingen",
+        "by_using": "Door onze website te gebruiken, gaat u akkoord met onze",
+        "agree_on_data_collection": "Door onze website te gebruiken, gaat u akkoord met het verzamelen van gegevens zoals beschreven in ons ",
+        "change_preferences": "U kunt uw voorkeuren op elk moment wijzigen",
+        "cancel_dialog_title": "Weet u zeker dat u uw instellingen wilt annuleren?",
+        "privacy_policy": "Privacybeleid",
+        "allow_category_tracking": "Tracking van [CATEGORY_NAME] toestaan",
+        "disallow_category_tracking": "Tracking van [CATEGORY_NAME] weigeren"
+    }
+}
```

Comment:
@BC-tymurbiedukhin should we add a new line at the end of file?

Reply:
updated.